deploy/cluster-manager/crds/0000_01_operator.open-cluster-management.io_clustermanagers.crd.yaml

```diff
@@ -37,6 +37,10 @@ spec:
             that govern registration and work distribution for attached Klusterlets.
           type: object
           properties:
+            imagePullSecret:
+              description: ImagePullSecret represents the secret to pull images of
```

Comment:
I'm not clear on the value of allowing a user to customize the name of the secret containing the image pull credentials.  Why don't we simply declare that it will be secret `Foo`?  The secret exists in a namespace we own, so this seems like an area of flexibility you will have to investigate for every bug that doesn't add power that a user needs.